docs/index.asciidoc

```diff
@@ -99,9 +100,29 @@ template.json:
  }
 --------------------------------------------------
 
-As illustrated above, through the use of 'opid', fields from the Logstash events can be referenced within the template.
+As illustrated above, through the use of 'opid', fields from the Logstash
+events can be referenced within the template.
 The template will be populated per event prior to being used to query Elasticsearch.
 
+Note that when using `query_template, the Logstash attributes `result_size`
+and `sort` will be ignored. They should be specified directly in the JSON
```

Comment:
Consider "Note that when **you use** the `query_template`, ..."  to give the sentence a subject.

Reply:
Thanks for the suggestion, will tweak accordingly and merge :-)